src/api/java/appeng/api/parts/IPartBakedModel.java

```diff
@@ -0,0 +1,15 @@
+
+package appeng.api.parts;
+
+
+import java.util.List;
+
+import javax.annotation.Nullable;
+
+import net.minecraft.client.renderer.block.model.BakedQuad;
+
+
+public interface IPartBakedModel
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Annotate the "IPartBakedModel" interface with the @FunctionalInterface annotation [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orod.org/coding_rules#rule_key=squid%3AS1609)